File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("naruto-character_sdk")

describe("NarutoCharacterSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# NarutoCharacter SDK exists test

require "minitest/autorun"
require_relative "../NarutoCharacter_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = NarutoCharacterSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
